docker/README.md

```diff
@@ -55,21 +52,26 @@ The following command launches a container configured with one of the public net
 ```bash
 docker run --rm -it \
     -p 4190:8080 \
+    -p 4191:7833 \
     -e NETWORK=mainnet \
     -e FAST_CATCHUP=1 \
     -e TELEMETRY_NAME=name \
     -e TOKEN=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa \
+    -e START_KMD=1 \
     -v ${PWD}/data:/algod/data/ \
     --name mainnet-container \
     algorand/algod:latest
 ```
 
 Explanation of parts:
 
-* `-p 4190:8080` maps the internal algod REST API to local port 4190
+* `-p 4190:8080` maps the internal algod REST API to local port 4190.
+* `-p 4191:7833` maps the internal kmd REST API to local port 4191.
 * `-e NETWORK=` can be set to any of the supported public networks.
-* `-e FAST_CATCHUP=` causes fast catchup to start shortly after launching the network.
 * `-e TELEMETRY_NAME=` enables telemetry reporting to Algorand for network health analysis. The value of this variable takes precedence over the `name` attribute set in `/etc/algorand/logging.config`.
+* `-e FAST_CATCHUP=1` causes fast catchup to start shortly after launching the network.
+* `-e START_KMD=1` signals to entrypoint to start the kmd REST API (THIS SHOULD NOT BE USED IN PRODUCTION).
+* `-e TELEMETRY_NAME=` enables telemetry reporting to Algorand for network health analysis.
```

Comment:
TELEMETRY_NAME listed twice
```suggestion
* `-e TELEMETRY_NAME=` enables telemetry reporting to Algorand for network health analysis. The value of this variable takes precedence over the `name` attribute set in `/etc/algorand/logging.config`.
* `-e FAST_CATCHUP=1` causes fast catchup to start shortly after launching the network.
* `-e START_KMD=1` signals to entrypoint to start the kmd REST API (THIS SHOULD NOT BE USED IN PRODUCTION).
```

Maybe all the bullets here should list the values? (i.e. 'name' and 'mainnet')